### PR TITLE
Add support for RISC-V

### PR DESCRIPTION
**Pull request by Willsparker**
Signed-off-by: William Parker william.parker1@ibm.com

### CODE REVIEW

File: src/org/openj9/envInfo/JavaInfo.java [hunk 1]
```diff
@@ -102,7 +104,7 @@ public String getJDKImpl() {
         impl = impl.toLowerCase();
         if (impl.contains("ibm")) {
             return "ibm";
-        } else if (impl.contains("openj9")) {
+        } else if (impl.contains("openj9") || impl.contains("OpenJ9")) {
```
Comment (llxia) [inline src/org/openj9/envInfo/JavaInfo.java]: `impl` is set to toLowerCase() at line 104. I do not think this change is needed.

File: src/org/openj9/envInfo/JavaInfo.java [hunk 2]
```diff
@@ -58,6 +58,8 @@ public String getSPEC() {
             spec += "_arm";
         } else if (osArch.contains("sparcv9")) {
             spec += "_sparcv9";
+	} else if (osArch.contains("riscv")) {
+	    spec += "_riscv";
```
Comment (llxia) [inline src/org/openj9/envInfo/JavaInfo.java]: Please fix the indentation

**Comment by llxia** [comment]
Could you sign off your commit? And do we have any existing issues that this PR can link to? Thanks

File: src/org/openj9/envInfo/JavaInfo.java (hunk 1 above)
Comment (Willsparker) [reply src/org/openj9/envInfo/JavaInfo.java]: I was seeing this message before I put the change in:
```
Cannot determine System.getProperty('os.arch')=riscv

System.getProperty('java.vm.name')=Eclipse OpenJ9 VM
```

File: src/org/openj9/envInfo/JavaInfo.java (hunk 2 above)
Comment (Willsparker) [reply src/org/openj9/envInfo/JavaInfo.java]: Fixed

**Comment by Willsparker** [comment]
ref: https://github.com/eclipse/openj9/issues/7421

File: src/org/openj9/envInfo/JavaInfo.java (hunk 1 above)
Comment (llxia) [reply src/org/openj9/envInfo/JavaInfo.java]: Please revert the above change and try again.

Comment (Willsparker) [reply src/org/openj9/envInfo/JavaInfo.java]: Ah, sorry, I've just seen it prints it before it's set to `toLowerCase()` and that'll be what I'm seeing.

**Pull request merged**